### ISSUE TITLE
Voiceover Braille editor accessibility issues

### ISSUE THREAD

**Issue by deathdogg** [microsoft/vscode#161673]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version:1.7
- OS Version:MacOS12.6

Steps to Reproduce:

1. Hook up a working braille display, I.E. a HIMS Polaris to VoiceOver and run VSCode.
2. You will notice that due to the paging behavior of VoiceOver, braille doesn't sync up properly on the display. I can be on line 1, for instance, and then scroll down to line 3, move my cursor, and the keyboard focus will remain at the point before I moved the cursor with the braille display. I can also move the keyboard focus with the arrow keys like normal, but although VoiceOver announces the correct up-to-date focus information, the braille display doesn't sync up with this info. For example, if I have 3 lines of code, then navigate to the middle of line 2, braille will actually display line 3 or 1, but never line 2 unless the keyboard focus is right at the beginning of that line. In other words, voiceover's speech, which exhibits correct behavior, doesn't sync up with what is being displayed with braille. This issue doesn't occur with any other code editor I use, probably because they use native editing APIs that work with Voiceover and braille out of the box. I don't see this issue occurring on windows. The braille editing experience on there is actually quite nice, so it's definitely specific to VoiceOver and MacOS.

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! It looks like you may be using an old version of VS Code, the latest stable release is 1.71.2. Please try upgrading to the latest version and checking whether this issue remains.

Happy Coding!

**Comment by deathdogg** [comment]
I just downloaded the new update and the issue persists.

**Comment by isidorn** [comment]
@deathdogg thanks for opening this issue. I believe this is a duplicate of https://github.com/microsoft/vscode/issues/95783
Which is actually an upstream issue https://bugs.chromium.org/p/chromium/issues/detail?id=1073074

I have just commented on that Chromium issue and it would be great if you comment as well. I have aslo reached out to our Google contact.
Closing this as a duplicate, and if you disagree do let me know. Thanks
